### DISCUSSION TITLE
PCB - File

### DISCUSSION THREAD

**Discussion by basten82** [kachurovskiy/nanoels#58] [General]
Hello,

could you share the PCB - Design file directly from jlcpcb, the easyeda file ? 
This would be nice.
So I can change the pin width for the screw terminals.

Thanks

**Comment by kachurovskiy** [comment]
@basten82 I've provided EasyEDA offline project files in https://github.com/kachurovskiy/nanoels/tree/main/h1/pcb/easyeda - did you try them? Thank you.

**Comment by basten82** [comment]
@kachurovskiy Perfect, thanks....!!!!
